vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -35,6 +35,14 @@ L.OSM.TransportMap = L.OSM.TileLayer.extend({
   }
 });
 
+L.OSM.OPNVKarte = L.OSM.TileLayer.extend({
```

Comment:
This file cames from upstream, doesn't it?

Reply:
It does but we define our own instead I think because we change the attribution relative to upstream?

Reply:
Yes, this is an upstream vendored file. The changes are already made upstream, this was a `vendorer update` where I just updated leaflet-osm.